organizer_notes/food.md

```diff
@@ -0,0 +1,9 @@
+# Food Resources
+
+## Genre Ideas
+Indian, greek/middle-eastern, pizza, thai, fajitas, Chipolte, chinese
+
+## Restaurants We Like
+(easy, reliable, good options)
+
+* wip
```

Comment:
@hayesmp can you add in the food vendors that you've used before and like?

Reply:
We use Foodee. The only restaurant I am aware of is Taj Palace for Indian.